app/models/miq_ae_method.rb

```diff
@@ -56,6 +56,12 @@ def namespace
     fqname.split("/")[0..-3].join("/")
   end
 
+  def data_for_expression
+    raise ArgumentError, "method is not an expression" if location != "expression"
```

Comment:
`ArgumentError` feels a little funny to me (since this method doesn't accept any arguments), what do you think of `TypeError`, `NotImplementedError` or `RuntimeError`?

Reply:
Agreed - I guess RuntimeError is the most accurate.